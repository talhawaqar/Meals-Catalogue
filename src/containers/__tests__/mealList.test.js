import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { createStore, applyMiddleware } from 'redux';
import { MemoryRouter } from 'react-router-dom';
import reduxThunk from 'redux-thunk';
import reducer from '../../reducers';
import MealList from '../MealsList';

it('renders correctly when store is provided', () => {
  const store = createStore(reducer, applyMiddleware(reduxThunk));
  const mealList = renderer.create(
    <Provider store={store}>
      <MemoryRouter initialEntries={['/']}>
        <MealList />
      </MemoryRouter>
    </Provider>,
  ).toJSON();
  expect(mealList).toMatchSnapshot();
});
