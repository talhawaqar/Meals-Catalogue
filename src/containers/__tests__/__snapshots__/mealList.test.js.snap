// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when store is provided 1`] = `
Array [
  <div
    className="hero-bg"
  >
    <h1>
      The meals you love
    </h1>
    <h2>
      Find meals that best suits your taste buds
    </h2>
  </div>,
  <div
    className="container"
  >
    <div
      className="row pt-5 pb-3"
    >
      <div
        className="col-12 filter"
      >
        <span>
           Select categories 
        </span>
        <select
          onChange={[Function]}
        />
      </div>
    </div>
    <div
      className="row"
    />
  </div>,
]
`;
